episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+  ggplot() +
+  geom_ribbon(
+    aes(
+      date,
+      ymin = severity_low,
+      ymax = severity_high,
+      fill = method
+    ),
+    alpha = 0.2, show.legend = FALSE
+  ) +
+  geom_line(
+    aes(date, severity_mean, colour = method)
+  ) +
+  geom_hline(data = rolling_cfr_adjusted_end,
+             aes(yintercept = severity_mean)) +
+  geom_hline(data = rolling_cfr_adjusted_end,
+             aes(yintercept = severity_low), lty = 2) +
+  geom_hline(data = rolling_cfr_adjusted_end,
+             aes(yintercept = severity_high), lty = 2)
+```
+
+The horizontal line represents the delay-adjusted CFR estimated at the outbreak's end. The dotted line means the estimate has a 95% confidence interval (95% CI).
+
+:::::::::::::::::::::::::::::::::::::::::::: challenge
+
+### Interpret the early-stage CFR estimate
+
+Based on the figure above:
+
+- How much difference in days is between the date in which the 95% CI of the estimated _delay-adjusted CFR_ vs _naive CFR_ cross with the CFR estimated at the end of the outbreak?
+
+:::::::::::::::::::::: hint
+
+We can use either visual inspection or analysis of the output data frames.
+
+::::::::::::::::::::::
+
+:::::::::::::::::::::: solution
+
+There is almost one month of difference.
+
+Note that the estimate has considerable uncertainty at the beginning of the time series. After two weeks, the delay-adjusted CFR approaches the overall CFR estimate at the outbreak's end.
+
+Is this pattern similar to other outbreaks? We can use the data sets in this episode's challenges. We invite you to find it out!
+
+::::::::::::::::::::::
+
+
+::::::::::::::::::::::::::::::::::::::::::::
+
+
+
+::::::::::::::::: spoiler
+
+### rolling or time-varying?
+
+The `{cfr}` package has a function called `cfr_time_varying()` with different functionality that makes them not comparable.
+
+#### `cfr_rolling()`
+
+`cfr_rolling()` shows the estimated CFR on each outbreak day, given that future data on cases and deaths is unavailable at the time. The final value of `cfr_rolling()` estimates is identical to `cfr_static()` on the same data.
+
+Remember, as shown above, `cfr_rolling()` is helpful to get early-stage CFR estimates and check whether an outbreak's CFR estimate has stabilised. Thus, `cfr_rolling()` is not sensible to the length or size of the epidemic.
+
+#### `cfr_time_varying()`
+
+On the other hand, `cfr_time_varying()` calculates the CFR over a moving window and helps to understand changes in CFR due to changes in the epidemic, e.g. due to a new variant or increased immunity from vaccination.
+
+However, `cfr_time_varying()` is sensible to sampling uncertainty. Thus, it is sensible to the size of the outbreak. The higher the number of cases with expected outcomes on a given day, the more reasonable estimates of the time-varying CFR we will get. But if we have a small number of cases throughout the epidemic, we probably need to rely on `cfr_rolling()` that uses the cumulative data.
```

Comment:
I am not completely sure about this sentence: "*But if we have a small number of cases throughout the epidemic, we probably need to rely on `cfr_rolling()` that uses the cumulative data."* 
		This seems incorrect to me given the reasoning, as all static CFR estimates are also likely to be biased by changes to the case definition or by changes in virulence or severe outcomes. I think those factors, rather than the size of the outbreak, should be the main consideration when picking a method. Not an epidemiologist, so could be wrong.

Reply:
I intended to rephrase this https://github.com/epiverse-trace/cfr/issues/123#issuecomment-1921364519 so now opted to add it more explicitly to avoid potential misunderstandings in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/0b250efd4945e0e65f2aefb30face60b5577d17d

Reply:
Okay thanks